packages/universal_profiling_symbolizer/changelog.yml

```
@@ -0,0 +1,6 @@
+# newer versions go on top
+- version: 8.8.0-preview
+  changes:
+    - description: Initial draft of the package
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/1
```

```suggestion
      link: https://github.com/elastic/integrations/pull/5879
```

This one should be updated